src/components/admin/admin-settings-tab.yml

```diff
@@ -1,7 +1,7 @@
 adminSettingsTab:
-  homepageInfoHeader:
-    en: Homepage info
-    de: Homepage-Info
+  announcementHeader:
+    en: Announcement
+    de: Benachrichtigung
```

Comment:
Maybe "Ankündigung" is better here. I also was thinking if we maybe should put the word "global" or "website" in front to make it clearer what it does?

Reply:
Makes sense, I changed to:
- [en] "Global announcement"
- [de] "Globale Ankündigung"

Reply:
Poifect :+1: